_layouts/default.html

```diff
@@ -3,9 +3,6 @@
   <head>
     <meta charset="UTF-8" />
 
-    <link rel="preconnect" href="https://fonts.gstatic.com" />
-    <link rel="preload" href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,400;0,700;1,400;1,700&display=swap" as="style" type="text/css" crossorigin />
```

Comment:
Dev tools had warnings about this both not working due to some cross-origin issue and not actually being used, so I removed it. (There were no other warnings)